src/core/views.py

```diff
@@ -866,7 +866,7 @@ def add_user(request):
     form = forms.EditAccountForm()
     registration_form = forms.AdminUserForm(active='add', request=request)
     return_url = request.GET.get('return', None)
-    role = request.GET.get('role', None)
```

Comment:
We are only assigning the role if another role is not defined in `request.GET["role"]`. This seems to contradict the commit title. Is this what we intended?

Reply:
Indeed! Commit incoming.